scripts/run-dev.mjs

```diff
@@ -42,7 +42,7 @@ function launchFrame({ shutdown }) {
 }
 
 async function run() {
-  const [env, server] = await Promise.all([prepareEnvironment(), launchDevServer()])
+  const [, server] = await Promise.all([prepareEnvironment(), launchDevServer()])
```

Comment:
I personally prefer the previous style, showing what's destructured in the response. thoughts @floating @Jamchello?